apps/pie-docs/src/patterns/messaging-and-chatbots/guidance/guidance.md

```diff
@@ -0,0 +1,541 @@
+
+Tails are a small extension that point towards the direction of the message’s sender that are attached to individual messages. They can vary in design depending on the third party platform that is being used.
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use tails where available with the third party platform to provide additional context to the user’s."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t apply tails to individual messages within a group, except for the last message in the sequence."
+        ]
+    }
+} %}
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/messaging-and-chatbots/messages-tails.svg",
+  alt: "",
+  width: "290px"
+} %}
+
+
+### Groups
+
+Messages sent in quick succession can be grouped to show users they are connected. Different third-party platforms may have varying time thresholds for defining grouped messages. 
+
+The spacing between individual messages is reduced to ensure they are visually tied together, and only the last message in a sequence retains its tail, while the others do not.
+
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/messaging-and-chatbots/groups-variants.svg",
+  variant: "secondary",
+  width: "240px",
+  alt: ""
+} %}
+
+---
+
+## Structured responses
+
+Structured responses are useful for chatbots that present choices to the user that are easy for the bot to understand, when selected, all structured responses will change their visual appearance and a user message will appear with the selected content.
+
+Structured responses are always paired with a message to provide context to the options provided.
+
+{% notification {
+  type: "information",
+  iconName: "info-circle",
+  message: "Ensure the background colour of the structured response chip matches that of the correspondent’s messages to visually pair them."
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Align the options to the side of the correspondent offering them to make it clear which side of the conversation they belong to.",
+            "Always provide an automated message response from the user with the option they selected for confirmation and visual feedback.",
+            "Always disable the options once a selection has been made to ensure a clear direction for the conversation."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Avoid structured responses for complex, emotional, ambiguous, or highly personalised interactions, as they can feel too rigid."
+        ]
+    }
+} %}
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/messaging-and-chatbots/content-types-structured-responses.svg",
+  alt: "",
+  width: "800px"
+} %}
+
+<h3> Interactive states </h3>
+
+Outlines the atomic level interactive elements for the component.
+
+{% contentLayout %}
+  {% contentItem %}
+  **Default**
```

Comment:
i think it's more semantic to use <h4> here, can you please check if that works fine? 

```suggestion
  <h4>Default</h4>
```

if so we need to update the references too.